terraform/variables.tf

```diff
@@ -0,0 +1,42 @@
+variable "app_name" {
+  description = "Application name"
+  type        = string
+}
+
+variable "model_name" {
+  description = "Model name"
+  type        = string
+}
+
+variable "channel" {
+  description = "Charm channel"
+  type        = string
+  default     = "latest/stable"
+}
+
+variable "revision" {
+  description = "Charm revision"
+  type        = number
+  nullable    = true
```

Comment:
Can't this line be omitted? https://developer.hashicorp.com/terraform/language/values/variables#disallowing-null-input-values

Reply:
Yes, `true` is default, but it's so weird to me that I though keeping it explicit would be valuable.